src/ocm/components/hosts/ClusterHostsTable.tsx

```diff
@@ -37,11 +38,11 @@ export interface ClusterHostsTableProps {
 const ClusterHostsTable: React.FC<ClusterHostsTableProps> = ({ cluster, skipDisabled }) => {
   const { onEditHost, actionChecks, onEditRole, actionResolver, ...modalProps } =
     useHostsTable(cluster);
-
+  const { t } = useTranslation();
   const content = React.useMemo(
     () => [
-      hostnameColumn(onEditHost, undefined, actionChecks.canEditHostname),
-      roleColumn(actionChecks.canEditRole, onEditRole, getSchedulableMasters(cluster)),
+      hostnameColumn(t, onEditHost, undefined, actionChecks.canEditHostname),
+      roleColumn(t, actionChecks.canEditRole, onEditRole, getSchedulableMasters(cluster)),
       statusColumn(AdditionalNTPSourcesDialogToggle, onEditHost, UpdateDay2ApiVipDialogToggle),
```

Comment:
```suggestion
      statusColumn(t, AdditionalNTPSourcesDialogToggle, onEditHost, UpdateDay2ApiVipDialogToggle),
```

Reply:
It's true. At first t was optional. I've changed it